packages/liveblocks-core/tsconfig.json

```diff
@@ -4,7 +4,8 @@
 
   "compilerOptions": {
     // TODO: Try to get rid of these overrides
-    "noUncheckedIndexedAccess": false // OVERWRITTEN to relax a bit
+    "noUncheckedIndexedAccess": false, // OVERWRITTEN to relax a bit
+    "types": ["vitest/globals"]
```

Comment:
It's not intuitive but setting `types` will stop TS from auto-importing types from `@types/*` packages: https://www.typescriptlang.org/tsconfig/#types. Maybe we could have a `.d.ts` file with `/// <reference types="vitest/globals" />` instead?

I would also be fine with the option to not use globals at all.

Reply:
I've just removed all magic globals indeed 👍